.env.example

```
@@ -41,4 +41,4 @@ S3_PUBLIC_URL=https://change-me.cloudfront.net
 # COMPOSE_PROFILES=
 # COMPOSE_DB_URL=jdbc:mysql://your-rds-endpoint.ap-northeast-2.rds.amazonaws.com:3306/moigo?serverTimezone=Asia/Seoul&characterEncoding=UTF-8&useSSL=true
 
-KAKAO_REST_API_KEY=8aae9d5d3ac91d8f760adcbccea8c890
\ No newline at end of file
+KAKAO_REST_API_KEY=change-me
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)